app/Http/Controllers/BookingFormController.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Appointment;
+use App\Client;
+use App\Treatment;
+use View;
+
+class BookingFormController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // return View::make('bookingform')->withTreatments(Treatment::all());
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {   
+        return View::make('bookingform')->withTreatments(Treatment::all());
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $this->validate($request, [
+            'treatmentType' => 'required',
+             'treatmentDate' => 'required',
+             'treatmentTime' => 'required',
+            'firstName' => 'required',
+            'lastName' => 'required',
+            'email' => 'required',
+            'tel' => 'required',
+            'houseNumber' => 'required',
+            'street' => 'required',
+            'town' => 'required',
+            'county' => 'required',
+            'postcode' => 'required',
+        ]);
+
+        $client = new Client;
```

Comment:
We have set the fillables on the model, meaning we can use the `create` method here
```php
$client = Client::create([
        'firstName'   => $request->input('firstName'),
        'lastName'    => $request->input('lastName'),
        'email'       => $request->input('email'),
        'tel'         => $request->input('tel'),
        'houseNumber' => $request->input('houseNumber'),
        'street'      => $request->input('street'),
        'town'        => $request->input('town'),
        'county'      => $request->input('county'),
        'postcode'    => $request->input('postcode'),
]);
```